src/views/Hero/index.view.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import PostcardStack from "components/PostcardStack/index.view";
+import EmailSubscriptionForm from "components/EmailSubscription/index.view";
+import "./Hero.scss";
+
+type HeroProps = {
+  pageName: string;
+  title: string;
+  description: string;
+};
```

Comment:
I am not married to this `pageName` prop, but this is essentially what we should be using to distinguish between what view we are on and what view to render.